NodePhantomJS/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM selenium/base:2.44.0
+MAINTAINER Selenium <selenium-developers@googlegroups.com>
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+#===================
+# Timezone settings
+# Possible alternative: https://github.com/docker/docker/issues/3359#issuecomment-32150214
+#===================
+ENV TZ "US/Pacific"
+RUN echo "US/Pacific" | sudo tee /etc/timezone \
+  && dpkg-reconfigure --frontend noninteractive tzdata
+
+#==============
+# PhantomJS
+#==============
+RUN apt-get update -y
+RUN apt-get install bzip2 curl libfreetype6 libfontconfig1  -y
```

Comment:
Do you really need to install curl? `wget` is already installed in the [base image](https://github.com/SeleniumHQ/docker-selenium/blob/master/Base/Dockerfile#L19) There should be no reason to install two different tools for downloading internet content.